resources/linux/debian/control.in

```diff
@@ -1,6 +1,6 @@
 Package: <%= appFileName %>
 Version: <%= version %>
-Depends: git, gconf2, gconf-service, libgtk2.0-0, libudev0 | libudev1, libgcrypt11 | libgcrypt20, libnotify4, libxtst6, libnss3, python, gvfs-bin, xdg-utils, libcap2
+Depends: git, gconf2, gconf-service, libgtk2.0-0, libudev0 | libudev1, libgcrypt11 | libgcrypt20, libnotify4, libxtst6, libnss3, python, gvfs-bin, xdg-utils, libcap2, libx11-xcb1, libxss1, libasound2, libxkbfile1
```

Comment:
I would change the `libasound2` dependency to have the same minimum version as the Chrome package: `libasound2 (>= 1.0.16)`.

Reply:
On that note, you _may_ want to add a minimum version to the following:
* `libnss3` -> `libnss3 (>= 2:3.22)`